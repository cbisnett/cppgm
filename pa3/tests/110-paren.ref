123
error
error
eof
