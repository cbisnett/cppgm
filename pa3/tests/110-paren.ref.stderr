ERROR: closing bracket expected in controlling expression
ERROR: expected identifier or literal in controlling expression
